front/app/containers/Admin/messaging/texting/SMSCampaignPreview/index.tsx

```diff
@@ -283,10 +305,15 @@ const TextMessagePreview = (props: WithRouterProps) => {
               icon="send"
               iconPos="right"
               disabled={sendCampaignButtonIsDisabled}
+              processing={isLoading}
             >
               Send Now
             </StyledModalButton>
           </Box>
+          {hasTooManySegmentsError && <Error text="Too many segments error" />}
+          {hasMonthlyLimitReachedError && (
+            <Error text="Monthly limits exceeded error" />
+          )}
```

Comment:
The text still needs to change + it's not possible to remove these errors once they're there because that didn't make sense to me.

Reply:
I think that's fine, though maybe we should keep the button disabled if one of these errors comes back. Neither of them can be fixed on this screen. Too many segments means going back to the edit page, and monthly limit exceeded can't be fixed other than by waiting or contacting an admin.

We could add more text to each error explaining what to do. "Too many segments in the message. Please return to the edit page and make the message shorter" and "You have reached your monthly limit of SMS messages to send. Please contact your site's admin for more info" or something.